client/src/views/SignUp.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import '../sass/LoginSignup.scss';
+import { Button, Form, Grid, Header, Message, Segment } from 'semantic-ui-react'
+
+const SignUpForm = () => (
+  <div className='signup-form'>
+    <Grid textAlign='center' style={{ height: '100%' }} verticalAlign='middle'>
```

Comment:
could you use a SCSS file instead of declaring the style directly? Same goes for all the other style declarations.